### DISCUSSION TITLE
Can I use this Extension

### DISCUSSION THREAD

**Discussion by ghazalialshafi** [VBAndCs/sVB-Small-Visual-Basic#32] [Ideas]
I know a very great extension of Small Basic which is LitDev it has hundreds of features can it be used with sVB and how I an archive it
https://github.com/litdev1/LitDev

**Comment by VBAndCs** [comment]
@allcpu 
No. Litdev targets the SmallBasicLibrary.dll but sVB targets the SmallVisualBasicLibrary.dll which makes them incompatible. I tried to fix this but failed. I think I can fix this by adding implicit conversions between the SmallVisualBasicLibrary.Primitive and SmallBasicLibrary.Primitive types, but this can wait. Actually, I don't see any benefit of using LitDev in sVB, as it already offers many advanced features in a simpler way.
